modules/gapi/src/backends/common/serialization.cpp

```diff
@@ -338,8 +338,13 @@ IIStream& operator>> (IIStream& is,       cv::gapi::wip::draw::Line &l) {
 
 IOStream& operator<< (IOStream& os, const cv::GCompileArg& arg)
 {
-    os << arg.tag;
-    arg.serialize(os);
+    ByteMemoryOutStream tmpS;
+    tmpS << arg.tag;
```

Comment:
Tag could be written to a primary stream.

Reply:
Thanks! Reworked this part!